src/components/molecules/NavBar/components/SoundCloudRadio/SoundCloudRadio.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback, useMemo, useState } from "react";
+import classNames from "classnames";
+
+import { generateId } from "utils/string";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./SoundCloudRadio.scss";
+
+export type SoundCloudRadioProps = {
+  station: string | undefined;
+  volume: number;
+};
+
+export const SoundCloudRadio: React.FC<SoundCloudRadioProps> = ({
+  station,
+  volume,
+}) => {
+  const id = useMemo(() => generateId("SoundCloudRadio-player"), []);
+
+  const [visible, setVisible] = useState(false);
```

Comment:
```suggestion
  const [isVisible, setVisible] = useState(false);
```

Lol, after 2 weeks of not touching things, I thought that's our new custom hook and thought like. Wow, such a cool name, why didn't I think of it before. We should definitely add the toggling functionality and remove our useToggler and then I remembered it's just the default hook :sadmancry

Reply:
The suggestion is counter to another style.

`visible` -> `setVisible` (clearer code)
`isVisible` -> `setIsVisible` (less clear code)

Reply:
Anyway, plenty of improvements for later on left. Can't fix all in current scope due to time constraints.